_posts/2018-09-21-first-class-resolved-file.md

```diff
@@ -0,0 +1,193 @@
+---
+layout: posts
+title: "First-class resolved files"
+authors:
+  - aehlig
+---
+
+In an earlier blog post, we described how a [resolved file can be used to freeze
+external dependencies](https://blog.bazel.build/2018/07/09/bazel-sync-and-resolved-file.html).
+
+- Repository rules may indicate how their arguments have to be changed
+  to reproduce identical output. This is the transition
+  from following a branch to a fixed commit. The Starlark
+  version of the `git_repository` rule already does that, and other rules
+  will follow suit soon.
+
+- With `bazel sync` there is a command to unconditionally fetch all external
+  repositories, and with the `--experimental_repository_resolved_file` option
+  all the reproducible descriptions can be collected in a file that contains
+  a Starlark value and hence is easy to work with later.
```

Comment:
How about "all the reproducible descriptions can be collected in a Starlark value that is written to a file"?